src/main/archetype/ui.frontend.forms/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "forms-archetype-sample",
+  "version": "1.0.0",
+  "description": "Sample for generating AF 2 Web Runtime clientlib",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\"",
+    "build:dev": "npx webpack --config webpack.dev.js",
+    "build:prod": "npx webpack --config webpack.prod.js && clientlib --verbose",
+    "start": "npx webpack serve --config webpack.dev.js --env mode=development"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/react": "^18.0.9",
+    "@types/react-dom": "^18.0.4",
+    "css-loader": "^6.7.1",
+    "css-minimizer-webpack-plugin": "^4.0.0",
+    "dotenv": "^16.0.1",
+    "html-webpack-plugin": "^5.5.0",
+    "mini-css-extract-plugin": "^2.6.0",
+    "postcss-loader": "^7.0.0",
+    "process": "^0.11.10",
+    "sass-loader": "^13.0.0",
+    "style-loader": "^3.3.1",
+    "ts-loader": "^9.3.0",
+    "typescript": "^4.6.4",
+    "webpack": "^5.72.1",
+    "webpack-bundle-analyzer": "^4.5.0",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "^4.9.0",
+    "webpack-import-glob-loader": "^1.6.3",
+    "webpack-merge": "^5.8.0"
+  },
+  "dependencies": {
+    "@adobe/exc-app": "^1.0.5",
+    "@adobe/react-spectrum": "^3.17.0",
+    "@aemforms/forms-core": "^0.11.0",
```

Comment:
You need to update dependencies as per `@adobe` scope

Reply:
use the latest version from https://www.npmjs.com/org/aemforms